react-frontend-app/src/App.js

```diff
@@ -32,42 +35,85 @@ function App() {
   }
 
   // Function that called when the submit button is pressed
-  function handleSubmit(event) {
+  async function handleSubmit(event) {
     event.preventDefault();
-    console.log('Submit was pressed');
-    console.log({surname}, {zipcode});
 
-    axios.get(`http://localhost:5000/surgeo?surname=${surname}&zipcode=${zipcode}`)
-      .then(res => console.log(res.data));
+    setVariable(await axios.get(`http://localhost:5000/surgeo?surname=${surname}&zipcode=${zipcode}`)
```

Comment:
Question: is it necessary to call setVariable twice? Isn't it possible to update variable by calling:

await axios.get(`http://localhost:5000/surgeo?surname=${surname}&zipcode=${zipcode}`).then(...)

?

Reply:
I might be wrong on this, so feel free to push back if I'm misinterpreting something

Reply:
Oh yes, this is a mistake from me — in theory, one would work — @kaylanhua I would just check that it still works if you remove one or the other.